frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/Maskinporten.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TabContent } from '../../TabContent';
+import { StudioButton, StudioHeading, StudioParagraph, StudioSpinner } from '@studio/components';
+import { useIsLoggedInWithAnsattportenQuery } from '../../../../../../../../hooks/queries/useIsLoggedInWithAnsattportenQuery';
+
+export const Maskinporten = (): React.ReactElement => {
```

Comment:
Nit and personal preference 😅 
```suggestion
export const Maskinporten = (): ReactElement => {
```